spec/factories/task_contributions.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :task_contribution do
+  end
```

Comment:
Can we add an association to a task here, so that the factory can be used just by calling?

```
create(:task_contribution)
```